plugins/feeds/public/urlhaus_payloads.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from datetime import timedelta
+
+from core import Feed
+from core.errors import ObservableValidationError
+from core.observables import Url, File, Hash
+
+
+class UrlhausPayloads(Feed):
+    default_values = {
+        "frequency":
+            timedelta(hours=1),
+        "name":
+            "UrlHausPayloads",
+        "source":
+            "https://urlhaus.abuse.ch/downloads/payloads/",
+        "description":
+            "URLhaus is a project from abuse.ch with the goal of sharing malicious URLs that are being used for malware distribution.",
+    }
+
+    def update(self):
+        for line in self.update_csv(delimiter=',', quotechar='"'):
+            self.analyze(line)
+
+    def analyze(self, item):
+
+        if not item or item[0].startswith("#"):
+            return
+
+        first_seen, url, filetype, md5, sha256, signature = item
+
+        context = {
+            'first_seen': first_seen,
+            'source': self.name
+        }
+
+        if url:
+            try:
+                url_obs = Url.get_or_create(value=url)
+                if signature != 'None':
+                    url_obs.tag(signature)
+                url_obs.add_context(context)
+                url_obs.add_source('feed')
+
+                context_malware = {
+                    'source': self.name
+                }
+
+                malware = File.get_or_create(value='FILE:{}'.format(sha256))
```

Comment:
if you're creating a file, I'd rather you call this variable `malware_file`

Reply:
Also, can't we create a proper `Malware` entity with the information we have here?